### DISCUSSION TITLE
XY Plot with random seed?

### DISCUSSION THREAD

**Discussion by caniyabanci76** [jags111/efficiency-nodes-comfyui#56] [Q&A]
Hi,

I have a basic XY plot workflow working seed against checkpoints.   When using the Seeds++ Batch node, it will increment N number of times from the initial seed number.    Is there a way for me to randomize the seed N number of times instead of it being an increment of the initial seed?  I have attached the workflow.
[xyplot_workflow.json](https://github.com/jags111/efficiency-nodes-comfyui/files/13628191/xyplot_workflow.json)

**Comment by metamountain** [comment]
Same Problem for me.... Have you solved the problem ?
